projects/Mallard/src/helpers/webview.ts

```diff
@@ -29,17 +29,16 @@ export const generateAssetsFontCss = (fontFamily: string) => {
 
 export const getBundleUri = (
     key: keyof typeof bundles,
-    use: 'auto' | 'dev' | 'prod' = 'auto',
+    use?: 'dev' | 'prod',
```

Comment:
I changed this too bc it felt cleaner than adding a default that is basically undefined but I can delet?